WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -38,21 +38,29 @@ final class OrderTableViewCell: UITableViewCell & SearchResultCell {
 
     /// Renders the specified Order ViewModel
     ///
-    func configureCell(viewModel: OrderDetailsViewModel, orderStatus: OrderStatus?) {
-        titleLabel.text = title(for: viewModel.order)
-        totalLabel.text = viewModel.totalFriendlyString
-        dateCreatedLabel.text = viewModel.formattedDateCreated
+    func configureCell(viewModel: OrderDetailsViewModel?, orderStatus: OrderStatus?) {
```

Comment:
nit: I was wondering if we could keep this function with non-optional `OrderDetailsViewModel` as before, and have another function when the view model is `nil` since it's an edge case?

Reply:
I'm not sure if that would be a good idea. I find that overloading methods can be confusing. Especially for `Optionals`. 

```swift
    func configureCell(viewModel: OrderDetailsViewModel?, orderStatus: OrderStatus?) {
 
    }

    func configureCell(viewModel: OrderDetailsViewModel, orderStatus: OrderStatus?) {

    }
```

I believe it sort of makes things harder to understand. 

---

I changed this again so that there's a `guard` and a `resetLabels()` function. This is kind of close to having a state machine. Let me know if that's better for reading or not? 489ab91b1ca44e74cf98bfb5de1b5c3272eafc4f

Reply:
the changes look great! 👍